network/a10/a10_server.py

```diff
@@ -31,6 +31,26 @@
 author: "Mischa Peters (@mischapeters)"
 extends_documentation_fragment: a10
 options:
+  host:
```

Comment:
Hi,
You don't need to document host, username, or password as the are documented in https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/module_docs_fragments/a10.py

Which is pulled in by `extends_documentation_fragment: a10`